guides/flex-layout.md

```diff
@@ -47,42 +47,52 @@ The `iron-flex-layout` component provides simple ways to use [CSS flexible box l
 *   Layout classes. The layout class stylesheet provides a simple set of class-based flexbox rules. Layout classes
     let you specify layout properties directly in markup.
 
-*   Custom CSS mixins.  The mixin stylesheet includes custom CSS mixins that can be applied 
-    inside a CSS rule using the `@apply` function. 
+*   Custom CSS mixins.  The mixin stylesheet includes custom CSS mixins that can be applied
+    inside a CSS rule using the `@apply` function.
 
-Using the classes or CSS mixins is largely a matter of preference. The following sections discuss 
+Using the classes or CSS mixins is largely a matter of preference. The following sections discuss
 how to use the each of the stylesheets.
 
-<aside><b>Note:</b> Before using either of these stylesheets, it's helpful to be familiar with the basics 
-of flexbox layout. Chris Coyier's [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is a 
+<aside><b>Note:</b> Before using either of these stylesheets, it's helpful to be familiar with the basics
+of flexbox layout. Chris Coyier's [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) is a
 good primer.</aside>
 
 ### Using layout classes
 
-To use layout classes import the `classes/iron-flex-layout` file.
+To use layout classes import the `iron-flex-layout-classes` file. You
+must do this in any element that uses any of the `iron-flex-layout` styles.
 
-    <link rel="import" href="bower_components/iron-flex-layout/classes/iron-flex-layout.html">
+    <link rel="import" href="bower_components/iron-flex-layout/iron-flex-layout-classes.html">
+
+Then include the module(s) that you need:
+
+    <style is="custom-style" include="iron-flex iron-flex-alignment">
```

Comment:
But you can't do a custom-style include without using a `<style is="custom-style">`, can you? You still need it to get the right module, so it's going to look identical in both cases